site/nginx.conf

```diff
@@ -21,10 +21,19 @@ http {
   ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
   ssl_prefer_server_ciphers on;
 
-  log_format combined_real_ip '$http_x_real_ip - $remote_addr - $remote_user [$time_local] '
-                              '$scheme "$request" $status $body_bytes_sent '
-                              '"$http_referer" "$http_user_agent"';
-  access_log /var/log/nginx/access.log combined_real_ip;
+  log_format json-log escape=json '{'
+   '"message":"$scheme $request done in ${request_time}s: $status",'
```

Comment:
let's move the status into a field. I want queries for status to be fast, I assume google indexes each field

Reply:
Honestly I'm not sure we need a message? Maybe just stick the `request` alone in there or put that in a field?

Reply:
Google extracts the `message` field as the info for the unexpanded log in the UI, so this is what you'd see while scrolling through the UI without expanding log entry. The status and request_time already have their own fields. `$request` is of the form "GET /path/to/thing HTTP/1.0" so I'm fine making that everything in `message`, perhaps also with the status

Reply:
For context, this matches the `AccessLogger` [message](https://github.com/hail-is/hail/blob/1215c5152e60825c5f7e1ab700cb6ba31788ecc2/hail/python/hailtop/hail_logging.py#L34)